redash/query_runner/athena.py

```diff
@@ -71,7 +71,7 @@ def configuration_schema(cls):
                     "title": "Schema Name",
                     "default": "default",
                 },
-                "glue": {"type": "boolean", "title": "Use Glue Data Catalog"},
```

Comment:
We need to think of an "upgrade path" for users with this setting enabled. One option is to read it in code and treat it same way as `schema_from_api` is true.